.github/workflows/format.yaml

```diff
@@ -0,0 +1,29 @@
+name: Formatting
```

Comment:
Are we planning to issue this PR upstream?  If so then may wish to remove the github workflows from this PR.

Reply:
It would require more work to support rng arg for the other backends, which we're not interested in. So, for now I'm not planning to push it upstream.